daemon/main.go

```diff
@@ -219,50 +219,77 @@ func checkMinRequirements() {
-	flags.StringVar(&socketPath, "socket-path", defaults.SockPath, "Sets the socket path to listen for connections")
-	flags.StringVar(&config.LBInterface, "lb", "",
-		"Enables load balancer mode where load balancer bpf program is attached to the given interface")
-	flags.BoolVar(&config.IPv4Disabled, "disable-ipv4", false, "Disable IPv4 mode")
-	flags.StringVar(&v4Prefix, "ipv4-range", "", "IPv4 prefix")
-	flags.StringVarP(&config.Tunnel, "tunnel", "t", "vxlan", "Tunnel mode vxlan or geneve, vxlan is the default")
-	flags.StringVar(&bpfRoot, "bpf-root", "", "Path to mounted BPF filesystem")
-	flags.String("access-log", "", "Path to access log of all HTTP requests observed")
-	flags.StringSlice("agent-labels", []string{}, "Additional labels to identify this agent")
-	flags.Bool("version", false, "Print version information")
-	flags.StringSliceVar(&loggers, "log-driver", []string{}, "logging endpoints to use")
-	flags.Var(option.NewNamedMapOptions("log-opts", &logOpts, nil), "log-opt", "log driver options for cilium")
+	flags.String(
+		"access-log", "", "Path to access log of all HTTP requests observed")
+	flags.StringSlice(
+		"agent-labels", []string{}, "Additional labels to identify this agent")
+	flags.StringVar(&config.AllowLocalhost,
+		"allow-localhost", AllowLocalhostAuto, "Policy when to allow local stack to reach local endpoints { auto | always | policy } ")
+	flags.StringVar(&bpfRoot,
+		"bpf-root", "", "BPF mount path filesystem")
+	flags.StringVar(&cfgFile,
+		"config", "", `Configuration file (default "$HOME/ciliumd.yaml")`)
+	flags.BoolP(
+		"debug", "D", false, "Enable debug messages")
+	flags.StringVarP(&config.Device,
+		"device", "d", "undefined", "Device to snoop on")
+	flags.BoolVar(&disableConntrack,
+		"disable-conntrack", false, "Disable connection tracking")
+	flags.BoolVar(&config.IPv4Disabled,
+		"disable-ipv4", false, "Disable IPv4 mode")
+	flags.StringVarP(&config.DockerEndpoint,
+		"docker", "e", "unix:///var/run/docker.sock", "Register a listener for docker events on the given endpoint")
+	flags.StringVar(&config.EnablePolicy,
+		"enable-policy", endpoint.DefaultEnforcement, "Enable policy enforcement")
+	flags.BoolVar(&enableTracing,
+		"enable-tracing", false, "Enable tracing while determining policy")
+	flags.StringVar(&v4Prefix,
+		"ipv4-range", "", "IPv4 prefix for endpoints, must be in correct format (10.N.0.1/16)")
+	flags.StringVar(&config.K8sEndpoint,
+		"k8s-api-server", "", "Kubernetes api address server (for https use --k8s-kubeconfig-path instead)")
+	flags.StringVar(&config.K8sCfgPath,
+		"k8s-kubeconfig-path", "", "Absolute path of the kubernetes kubeconfig file")
+	flags.BoolVar(&config.KeepConfig,
+		"keep-config", false, "When restoring state, keeps containers' configuration in place")
+	flags.BoolVar(&config.KeepTemplates,
+		"keep-templates", false, "Do not restore BPF template files from binary")
```

Comment:
Just renamed it. If somebody complains we can add a deprecated version. We haven't declared our CLI stable yet.